client/src/app/site/pages/meetings/pages/mediafiles/modules/mediafile-list/components/mediafile-list/mediafile-list.component.html

```diff
@@ -20,10 +20,9 @@ <h2>{{ 'Files' | translate }}</h2>
     </div>
 
     <!-- Multiselect info -->
-    <div *ngIf="isMultiSelect" class="central-info-slot">
```

Comment:
Is this intended?

Reply:
Yes, 
I tested if this changes the OpenSlides behaviour and saw nothing
And I would assume that the code should ideally be as similar as possible in the <!-- Multiselect info -->